addons/website_event_booth_exhibitor/views/event_booth_category_views.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo><data>
+
+    <record id="event_booth_category_view_form" model="ir.ui.view">
+        <field name="name">event.booth.category.view.form.inherit.website.event.booth.exhibitor</field>
+        <field name="model">event.booth.category</field>
+        <field name="inherit_id" ref="event_booth.event_booth_category_view_form"/>
+        <field name="arch" type="xml">
+            <group name="details" position="after">
+                <group string="Sponsorship" name="sponsor">
+                    <field name="use_sponsor"/>
```

Comment:
Could be great to add those fields to search view in order to ease search and group_by. What do you think ?